modules/insights-operator-one-time-gather.adoc

```diff
@@ -19,7 +19,7 @@ You must run a gather operation to create an Insights Operator archive.
 +
 [source,yaml]
 ----
-include::https://raw.githubusercontent.com/openshift/insights-operator/release-4.8/docs/gather-job.yaml[]
+include::https://raw.githubusercontent.com/openshift/insights-operator/release-4.9/docs/gather-job.yaml[]
```

Comment:
@xJustin, I'm not familiar with this repo. Is there any reason to expect that this file won't be available at OCP GA?

Reply:
This 4.9 file will be available at GA. Your change LGTM. Thanks for the catch!

Reply:
Thanks for confirming!